### ISSUE TITLE
Reviews truncated

### ISSUE THREAD

**Issue by milekpl** [cognitive-metascience/review_crawler#1]
Apparently, there's a problem in parsing HTML reviews:

https://pubpeer.com/publications/25353AAFD4FC52E2BEC8C7AD08B259#

@x-j Can you take a look? I guess it may be a problem of removing some HTML tags for lists.

**Comment by x-j** [comment]
Yes, my mistake, I forgot about 'ordered lists'.

[Line 115](https://github.com/cognitive-metascience/review_crawler/blob/f34b64737e341df60d27e718da699590b8b9bccc/crawling/crawling/spiders/mdpi_review_spider.py#L115) of mdpi review spider should read:

```
        for p in response.css('div.abstract_div p, ul, ol'):
```

I tested it on the provided example and it worked for this case. I would upload the fix but unfortunately I don't think I have permissions to push to this repo.

**Comment by milekpl** [comment]
@x-j Changed the permissions. See if you can push the fix.

This, unfortunately, means that we should update the corpus (at least for reviews that are HTML-based). This is why I thought maintaining an index in an SQL database would be a good idea. Anyway, can we try to select these files for further processing without scraping everything once again?

**Comment by x-j** [comment]
This issue is solved in v2.1 of the corpus.
